app/templates/pages.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="ui segment">
+  <div class="ui grid">
+    <div class="sixteen wide column">
+      <h1 class="ui header">{{model.name}}</h1>
+      <p>{{t 'Page URL: '}}<a href="{{model.url}}">{{model.url}}</a></p>
```

Comment:
@sumedh123 Check how it is implemented on eventyay use https://eventyay.com/imprint/ for reference

Reply:
@sumedh123 Showing the URL on the page does not make sense. do not show it on page.